core/templates/pages/topic-viewer-page/practice-tab/practice-tab.directive.html

```diff
@@ -1,31 +1,68 @@
 <div class="oppia-practice-tab-container">
-  <md-card class="oppia-practice-tab-card">
-    <button class="oppia-new-practice-session" ng-click="$ctrl.newPracticeSession()">
-      New Practice Session
-    </button>
+  <md-card class="oppia-page-card oppia-long-text">
+    <h2 class="tab-title">
+      <span translate="I18N_TOPIC_VIEWER_MASTER_SKILLS"></span>
+    </h2>
+    <span class="practice-content">
+      <span translate="I18N_TOPIC_VIEWER_SELECT_SKILLS"></span>
+    </span>
+    <md-checkbox ng-repeat="subtopic in $ctrl.availableSubtopics" ng-model="$ctrl.selectedSubtopicIndices[$index]">
+      <[subtopic.getTitle()]>
+    </md-checkbox>
+    <md-button ng-disabled="$ctrl.isStartButtonDisabled()"
+               ng-click="$ctrl.openNewPracticeSession()"
+               class="practice-session-button md-accent md-raised">
+      <span translate="I18N_TOPIC_VIEWER_START_PRACTICE"></span>
+    </md-button>
   </md-card>
 </div>
 <style>
-  .oppia-practice-tab-container {
-    margin-top: 20px;
+  practice-tab .oppia-long-text {
+    /* Since the same class is used in all the other tabs, it is defined in topic-viewer-page.directive.html,
+       but since those CSS classes are evaluated after the classes in this directive,
+       without the !important tag, any styles set here will be ignored for common classes.
+    */
+    width: 37.5vw !important;
+  }
+
+  @media (max-width: 1024px) {
+    practice-tab .oppia-long-text {
+      width: 60vw !important;
```

Comment:
Same thing here and below, try to avoid !important

Reply:
Done